test/simpleGaMultiSigTest.js

```diff
@@ -38,45 +45,40 @@ describe('SimpleGAMultiSig', () => {
   }
   let consensusInfo;
 
-  const getTxHash = (rlpTransaction) => new Uint8Array(Crypto.hash(Buffer.concat([
-    Buffer.from(aeSdk.getNetworkId()),
-    TxBuilderHelper.decode(rlpTransaction, 'tx'),
+  const getTxHash = async (rlpTransaction) => new Uint8Array(hash(Buffer.concat([
```

Comment:
@davidyuk I think we should provide a helper method for that. or does it already exist meanwhile? it doesn't seem to be convenient to get the tx hash right now

Reply:
yep, it is called `buildAuthTxHash`, available since sdk@12 https://aeternity.com/aepp-sdk-js/v12.0.0/api/classes/AeSdk.html#buildAuthTxHash
```suggestion
  const getTxHash = async (encodedTx) => aeSdk.buildAuthTxHash(encodedTx)
```

Reply:
perfect :)